workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/GetAppModal/TrialTimeline.tsx

```diff
@@ -27,57 +27,29 @@ export function TrialTimeline() {
 		'December',
 	];
 
-	const date = new Date();
-
-	date.setTime(date.getTime() + 30 * 24 * 60 * 60 * 1000);
-
-	const endOfTrialDay = date.getDate();
-	const endOfTrialMonth = months[date.getMonth()];
-
-	return (
-		<div className="get-app-modal-trial">
-			<div className="get-app-modal-trial-timeline">
-				<img
-					alt="circle fill"
-					className="get-app-modal-trial-timeline-icon-selected"
-					src={radioSelected}
-				/>
-
-				<img
-					alt="timeline"
-					className="get-app-modal-trial-timeline-dash"
-					src={timeline}
-				/>
-
-				<img
-					alt="circleFill"
-					className="get-app-modal-trial-timeline-icon"
-					src={circleFill}
-				/>
-			</div>
-
-			<div className="get-app-modal-trial-messages">
-				<div className="get-app-modal-trial-messages-item">
-					<div className="active title">
-						Today - free trial for 30 days.{' '}
-					</div>
-
-					<div className="description">
-						Start your free trial and test the full app for 30 days.
-					</div>
-				</div>
-
-				<div className="get-app-modal-trial-messages-item">
-					<div className="title">
-						{endOfTrialMonth} {endOfTrialDay}
-					</div>
-
-					<div className="description">
-						Your trial ends and this app will no longer function in
-						your project, unless you've subscribed during the trial.
-					</div>
-				</div>
-			</div>
-		</div>
-	);
+  return (
+    <div className="trial-timeline">
+      <p className="mb-2 trial-info">
+        Need help with license calculations?
+        <span className="info-button">
+          More Info
+          <CardClayIcon icon="question-circle-full" />
+        </span>
+      </p>
+      <CardButton
+        description="Trial licenses are intended for you to try the app before you buy. Typical trials are 30 days."
+        disabled={false}
+        icon={
+          <CardClayIcon className="trial-card-icon" icon="percentage-symbol" />
+        }
+        iconLeft={false}
+        onClick={() => {
+          setHandleSelected(true);
+          onSelectLicense(true);
+        }}
+        selected={handleSelected ? true : false}
+        title="Trial License"
+      />
+    </div>
```

Comment:
We already have the <ClayCard/> component. Please refactor this using ClayCard. You can use the 'active' _className_ plus the onClick().